src/Strategy/Node/Npm/PackageLock.hs

```diff
@@ -232,14 +241,10 @@ buildGraph packageJson directSet (WorkspacePackageNames workspacePackages) =
                 label requiredPkg (NpmDepVertexEnv EnvDevelopment)
           )
           depRequires
-      -- Add edge from parent
```

Comment:
We would want to capture dependencies only in the `requires` block, but this made it such that edges were created between `dependencies`'s children. This is still a spooky change!